pl_examples/pytorch_ecosystem/pytorch_geometric/poetry.lock

```diff
@@ -0,0 +1,1637 @@
+[[package]]
```

Comment:
i don't think that this super long lock file is needed... :]

Reply:
Yes, you are right ! I was going to remove it.